sonic_installer/main.py

```diff
@@ -111,36 +185,50 @@ def get_container_image_id_all(image_name):
     image_id_all = set(image_id_all)
     return image_id_all
 
+
 def hget_warm_restart_table(db_name, table_name, warm_app_name, key):
     db = SonicV2Connector()
     db.connect(db_name, False)
     _hash = table_name + db.get_db_separator(db_name) + warm_app_name
     client = db.get_redis_client(db_name)
     return client.hget(_hash, key)
 
+
 def hdel_warm_restart_table(db_name, table_name, warm_app_name, key):
     db = SonicV2Connector()
     db.connect(db_name, False)
     _hash = table_name + db.get_db_separator(db_name) + warm_app_name
     client = db.get_redis_client(db_name)
-    return  client.hdel(_hash, key)
+    return client.hdel(_hash, key)
+
+
+def print_deprecation_warning(deprecated_cmd_or_subcmd, new_cmd_or_subcmd):
+    click.secho("Warning: '{}' {}command is deprecated and will be removed in the future"
```

Comment:
>{} [](start = 31, length = 2)

extra pair?

Reply:
Oh, I see it.

---
In reply to: [453998034](https://github.com/Azure/sonic-utilities/pull/983#discussion_r453998034) [](ancestors = 453998034)

Reply:
No. It's intentional.